lib/index.js

```diff
@@ -158,21 +158,33 @@ class Linter {
   static errorsToMessages(filePath, errors, options) {
     errors = errors || [];
     options = options || {
-      verbose: false
+      verbose: false,
+      quiet:   false
     };
 
     if (errors.length === 0) {
       return '';
     }
 
-    let errorsMessages = errors.map((error) => this._formatError(error, options)).join('\n');
+    let errorsMessages = errors
+      .map((error) => this._formatError(error, options))
+      .filter(error => error !== '')
+      .join('\n');
+
+    if (errorsMessages === '') {
+      return '';
+    }
 
     return `${chalk.underline(filePath)}\n${errorsMessages}\n`;
   }
 
   static _formatError(error, options) {
     let message = '';
 
+    if (options.quiet && error.severity === WARNING_SEVERITY) {
```

Comment:
I know that it doesn't matter just yet, but I think conceptually we want to say that quiet means "only show errors". Can we make this conditional reflect that (e.g. `options.quiet && error.severity < ERROR_SEVERITY`)?